content/pt-br/docs/tasks/configure-pod-container/create-hostprocess-pod.md

```diff
@@ -0,0 +1,245 @@
+---
+title: Crie um Pod de Hostprocess do Windows
+content_type: task
+weight: 20
+min-kubernetes-server-version: 1.23
+---
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.23" state="beta" >}}
+
+Os contêineres de HostProcess do Windows permitem que você execute
+cargas de trabalho em um host Windows contêinerizado. Esses contêineres operam como
+processos normais, mas tem acesso à rede host do namespace,
+armazenamento e dispositivos, quando recebem os privilégios de usuário apropriados.
+Os contêineres HostProcess podem ser usados para implantar plugins de rede,
+configurações de armazenamento, plugins de dispositivos, kube-proxy, e outros
+componentes para nós do Windows, sem a necessidade de *proxies* dedicados ou
+a instalação direta dos serviços de host.
+
+Tarefas administrativas, como a instalação de `patches` de segurança, 
+coleta de logs de eventos, e mais pode ser realizado sem a necessidade 
```

Comment:
"podem ser realizadas" instead of "pode ser realizado" because "Tarefas administrativas podem ser realizadas"